package.json

```diff
@@ -135,22 +135,18 @@
   },
   "lint-staged": {
     "projects/canopy/src/**/*.svg": [
-      "npm run build:icons",
-      "git add"
```

Comment:
Ah good spot, it seems to be a mixed bag. The 'ts' files worked and the changes were committed, but the SVGs and SCSS files were updated and not added to the commit. It would be good to fix this at some stage. For now I've removed the commit from this PR.